File: C#/백준/Bronze/26264. 빅데이터？ 정보보호！/빅데이터？ 정보보호！.cs
using System;

namespace q{
    class w{
        static void Main(String[] args){
            int a = int.Parse(Console.ReadLine());
            String s = Console.ReadLine();
            int l = s.Length;
            if (l-7.5*a>0){
                Console.WriteLine("security!");
            }
            else if(l-7.5*a==0){
                Console.WriteLine("bigdata? security!");
            }
            else{
                Console.WriteLine("bigdata?");
            }
        }
    }
}

File: Java/백준/Bronze/26264. 빅데이터？ 정보보호！/빅데이터？ 정보보호！.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        double a = in.nextInt();
        String s = in.next();
        double n = s.length()-a*7.5;
        if (n==0){
            System.out.println("bigdata? security!");
        }
        else if(n>0){
            System.out.println("security!");
        }
        else{
            System.out.println("bigdata?");
        }
    }
}